### ISSUE TITLE
Interactive drag drop reorder in GridView

### ISSUE THREAD

**Issue by jiaohaibin** [flutter/flutter#46430]
Hope GridView can support interactive drag drop reorder.

**Comment by brightseagit** [comment]
Yes a reorderable gridview is required. For example the Google Keep Notes App.

**Comment by OliverRhyme** [comment]
Really need this as other libraries are either buggy or lacking of features.

**Comment by brightseagit** [comment]
@OliverRhyme  Currently, There have been major improvements and fixes in [Reorderable_grid_view](https://pub.dev/packages/reorderable_grid_view). 
Note: I'm talking about the latest Prerelease versions of the plugin.
This plugin is currently working great for me so you might also wanna give it a try.

**Comment by OliverRhyme** [comment]
> @OliverRhyme  Currently, There have been major improvements and fixes in [Reorderable_grid_view](https://pub.dev/packages/reorderable_grid_view). 
> Note: I'm talking about the latest Prerelease versions of the plugin.
> This plugin is currently working great for me so you might also wanna give it a try. 

Yes I've been using that instead. But I've created some modifications to be able to use the lazy builder and also having the ability of setting maxExtent instead of a fixed no. of columns

**Comment by idish** [comment]
Indeed [Reorderable_grid_view](https://pub.dev/packages/reorderable_grid_view) library looks pretty neat, though still missing out some important features (itemBuilder, slivers support).

**Comment by sososdk** [comment]
You can try [reorderable_plus](https://pub.dev/packages/reorderable_plus) .

**Comment by satvikpendem** [comment]
@sososdk how does it compare to [reorderable_grid_view](https://pub.dev/packages/reorderable_grid_view)?

**Comment by MhmdFouda** [comment]
Iam working on app clone google keep 
I didn't find any pakage help to get reorder able grid with children they have different size ![Screenshot_2023-02-22-14-38-31-457_com.fouda.mynote.jpg](https://user-images.githubusercontent.com/8207775/220661881-401b0813-e2ac-4a78-b708-2339cff7a827.jpg)

**Comment by tsquillario** [comment]
An official `ReordableGridView` would be a great addition. One that support `InteractiveViewer`.

**Comment by bgenidy** [comment]
> reorderable_plus

This package is awesome! thanks for sharing it, hope it picks traction and more of the community/flutter team can support making this as part of the core flutter framework
